mesh/mesh.go

```diff
@@ -473,19 +506,7 @@ func (msh *Mesh) logStateRoot(layerID types.LayerID) {
 
 func (msh *Mesh) calcValidLayerHash(layer *types.Layer) types.Hash32 {
```

Comment:
add comment that layer hash is aggregated hash

Reply:
done